src/provision/freedom-module.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "provision",
+  "description": "Provisions VMs on Digital Ocean",
```

Comment:
Yes, I agree with you. I needed to wrap the code in a freedom.js module for the sample app in order to use core.oauth. Plus, the SSH keygen will block for seconds, so I found this useful for that reason too. If you want to just link this into another module, that works too